respy/shared.py

```diff
@@ -26,37 +27,91 @@ def aggregate_keane_wolpin_utility(
     return value_function, flow_utility
 
 
-def create_base_draws(shape, seed):
-    """Create the relevant set of draws.
+def create_base_draws(shape, seed, monte_carlo_sequence):
+    """Create a set of draws from the standard normal distribution.
 
-    Handle special case of zero variances as this case is useful for testing.
-    The draws are from a standard normal distribution and transformed later in
-    the code.
+    The draws are either drawn randomly or from quasi-random low-discrepancy sequences,
+    i.e., Sobol or Halton.
+
+    `"random"` is used to draw random standard normal shocks for the Monte Carlo
+    exercises or because individuals face random shocks in the simulation.
+
+    `"halton"` or `"sobol"` can be used to change the sequence for two Monte Carlo
```

Comment:
@rafaelsuchy , please add the proper references for the integration strategies that we implement here as we did in a different context in `def transform_base_draws_with_cholesky_factor`.

Reply:
Postponed.